apps/framework-cli/src/cli/routines/metrics_console/run_console/app.rs

```diff
@@ -53,7 +72,43 @@ impl App {
         }
     }
 
-    pub fn req_per_sec(&mut self, new_total_requests: f64) {
+    pub fn req_per_sec(&mut self, new_total_requests: f64, path_metrics: &Vec<PathMetricsData>) {
         self.requests_per_sec = new_total_requests - self.total_requests;
+
+        // Initializes variables and vec for each path in summary to keep unwraps in ui.rs safe
```

Comment:
In general, convention across files will create bugs when someone else comes and changes things. People should be able to assume that they can change things locally to a function and that it won't break things in other places. Here this convention breaks that principle.

It would have been ok, in the same function to have something that assumes you can unwrap and makes sure that's safe. But that definitely should not be done cross files. That's what the types are here for.